src/App/Fossa/ProjectInference.hs

```diff
@@ -104,7 +105,7 @@ inferDefault dir = sendIO $ do
   tmp <- getTempDir
   writeFile (fromAbsDir tmp FP.</> ".fossa.revision") (show time)
 
-  pure (InferredProject (T.pack name) (T.pack (show time)) "master")
+  pure (InferredProject (T.pack name) (T.pack (show time)) Nothing)
```

Comment:
We previously hard-coded `master` here.  Is this safe to remove?

Reply:
Yes 👍